app/javascript/components/App.js

```diff
@@ -53,6 +53,8 @@ export default class App extends React.Component {
       current_user
     } = this.props
 
+    console.log(current_user)
```

Comment:
Dirty console log remains D:

Reply:
Haha sorry, we added it back to test something and forgot again. Gone now!